### PR TITLE
Added Sacramento file

### PR DESCRIPTION
**Pull request by pdeleon-ctg**
This is the beginning of the Sacramento file ☀️ .

Please provide feedback on:
- If you've been to Sacramento, what did you like/dislike?
- If you've never been to Sacramento, what interests you about coming here?
- Are you interested in learning about lodging? What do you prefer? Hotels? AirBNBs? etc.

### CODE REVIEW

File: pdeleon-ctg-sacramento.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Sacramento, CA
+
+### Restaurants
+- Shoki Ramen House
```
Comment (sathya-ghub) [inline pdeleon-ctg-sacramento.md]: What would you recommend here?!

Comment (pdeleon-ctg) [inline pdeleon-ctg-sacramento.md]: > What would you recommend at Shoki Ramen House?!

Tan Tan Men. It's their most popular dish!

**Comment by sathya-ghub** [comment]
@pdeleon-ctg Looks good to :ship: !

**Comment by pdeleon-ctg** [comment]
Thanks! 😄

**Pull request merged**